spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -372,36 +347,26 @@ def check_response_error(code)
   # POST
 
   context 'POST /authorship' do
+    let(:http_request) { post '/authorship', request_data.to_json, headers }
+
     context 'success' do
-      let(:http_request) do
-        post '/authorship', request_data.to_json, headers
-        expect(response.status).to eq(201)
-      end
+      after(:example) { expect(response.status).to eq(201) }
+
+      it_behaves_like 'it updates an existing contribution'
 
       context 'with sul_author_id' do
         let(:author_hash) { sul_author_hash }
-        it_behaves_like 'it creates new contributions and publications'
-        # TODO: modifies existing contributions.
+        it_behaves_like 'it creates new contributions and publications' # TODO: modifies existing contributions.
       end
 
       context 'with cap_profile_id' do
         let(:author_hash) { Hash[cap_profile_id: author.cap_profile_id] }
-        it_behaves_like 'it creates new contributions and publications'
-        # TODO: modifies existing contributions.
+        it_behaves_like 'it creates new contributions and publications' # TODO: modifies existing contributions.
       end
 
-      context 'to update a contribution' do
-        let(:request_data) { update_authorship_for_pub_with_contributions }
-        it_behaves_like 'it updates an existing contribution'
-      end
```

Comment:
Seems like this one changed the request_data before but now it's using all the same request data as the others.

Reply:
It moved above to R355 and didn't require any special `request_data` context.